spec/features/people_spec.rb

```diff
@@ -10,9 +10,24 @@
 
   end
 
+  scenario 'try to add a person with a duplicate identification' do
+    person = build(:person, identification: '10373737')
+
+    2.times do
+      expect(create_person(person))
+    end
+
+    expect(page).to have_content 'Identification has already been taken'
+  end
+
   scenario 'edits a person' do
-    create(:person)
-    update_person
+    person = create(:person)
+    visit people_path
+    click_link("Edit", match: :first)
+
+    fill_fields(person)
```

Comment:
Once the codebase grows we'll have a lot of fill_fields helper on different macros, so it's better to have a explicit name like `fill_person_form_fields` to reveal better the intention.

Sorry I'm annoying :sweat_smile: